cdk/setup-tool

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+
+MY_VERSION="1.0"
+MY_NAME="acmne-setup-tool"
```

Comment:
nit: personally I would not use the `MY_` prefix, but something like `TOOL_` or none at all